pkg/install/install.go

```diff
@@ -50,6 +60,11 @@ func (obj *installer) install(
 	ctx context.Context,
 	pkg *v1alpha1.Package,
 ) (*v1alpha1.Package, error) {
+	if obj.dryRun {
+		obj.status.SetStatus(fmt.Sprintf("Dry run: Simulating installation of %v...", pkg.Name))
+		time.Sleep(6 * time.Second)
```

Comment:
Again, please remove this `Sleep`.

Additionally, I think it would be great if we could take advantage of the kubernetes APIs dry-run feature, which can be enabled by setting the property in the create options of the request. However, our handwritten `PackageClient` does not support these `CreateOptions` yet. Would you be interested in adding this feature to the client and then adapting your PR to use it?

You may take the official kubernetes client as a reference: https://github.com/kubernetes/client-go/blob/52e5651101edcb2ecd8463ffdc281053cf6e63d4/kubernetes/typed/apps/v1/deployment.go#L123-L134

Reply:
Yeah I can do it. Will commit the changes. Once done.

Reply:
Awesome, you rock :guitar: